packages/validator/test/cli-validator/mock-files/warn-threshold.yml

```diff
@@ -0,0 +1,204 @@
+swagger: "2.0"
```

Comment:
Added this new API definition by copying one of the existing ones, then I removed a bunch of descriptions to cause enough warnings to allow the threshold-validator tests to pass.